### ISSUE TITLE
Local scheme copy for CI

### ISSUE THREAD

**Issue by KillerJulian** [Diizzayy/nuxt-graphql-client#505]
### Your use case

Locale schema file that Pipeline/Workflow/CI/CD does not fail if the endpoint cannot be reached.

### The solution you'd like

Option to store the fetched schema file in a local configured directory.

**Comment by KillerJulian** [comment]
By the way. The docs are incorrect.

```ts
export default defineNuxtConfig({
  modules: ['nuxt-graphql-client'],

  runtimeConfig: {
    public: {
      'graphql-client': {
        clients: {
          default: {
            host: '', // REQIURED "host"
            schema: '<relative_path_to_schema_file>'
          }
        }
      }
    }
  }
});
```
_https://nuxt-graphql-client.web.app/advanced/codegen#local-schema_

**Comment by LePtiDev** [comment]
Any news about this issue ? I got the same issue and i would like to have my schema on my repo to test it

**Comment by JulianWowra** [comment]
I am currently not aware of a fix for the problem, except to provide the schema locally via copy and paste or to have it built directly from the backend into a corresponding directory, as in my case.

**Comment by LePtiDev** [comment]
I have an idea but i need to do a PR but somebody can accepte it ? Codegen has a param to give you the choice of where you want to generate the types. If i expose this params to the nuxt config you will be able to have the types in the folder and after in the pipline to test your types. What do you think ?

**Comment by JulianWowra** [comment]
A similar approach has also occurred to me. Whether it is the best approach is certainly open to debate. I do think that you can make a pull request. However, it may take longer for it to be merged. I think the maintainer is unfortunately not that active for this project

**Comment by LePtiDev** [comment]
Yes i think the same. I have some issues in production he told me he gonna resolve it but he do nothing. Do you have a similar module in mind ? I like the type generation and the auto import. We can fork it and resolve it what do you think ?

**Comment by JulianWowra** [comment]
I currently do not have the capacity to create a fork of this project or contribute improvements, but feel free to improve the project. hopefully the maintainer will become more active again or hand over the project
